### ISSUE TITLE
Floating point exception (core dumped) for the resnet50-v2 model

### ISSUE THREAD

**Issue by ravishk1** [pytorch/glow#5388]
I tried to compile the model (resnet50-v2-7.onnx) which I downloaded from [ONNX-ZOO](https://github.com/onnx/models/tree/master/vision/classification/resnet/model). When I just compiled the model, by command
_./model-compiler -backend=CPU -model=resnet50-v2-7.onnx -emit-bundle/ -bundle-api=dynamic_
It runs perfectly fine, but when I compiled it after creating the profile, by command

_./image-classifier cat_285.png -image-mode=0to1 -model=resnet50-v2-7.onnx -dump-profile=profile.yml -model-input-name=data_
 _./model-compiler -backend=CPU -model=resnet50-v2-7.onnx -emit-bundle=bundle2/ -bundle-api=dynamic -load-profile=profile.yml -relocation-model=pic_

It produces an error, **Floating point exception (core dumped)**

What can be the problem? It's clear that the profiler is creating the problem in some node, but how this error can be resolved?

**Comment by jfix71** [comment]
Hi @ravishk1, It’s hard to say what the problem could be without more details. Can you compile in debug mode/run in a debugger to get a stack trace?

**Comment by wangshankun** [comment]
-keep-original-precision-for-nodes=Add,Sub,Mul,Div,Exp,SoftMax

**Comment by wangshankun** [comment]
-keep-original-precision-for-nodes=Div
